src/wasm-as.cpp

```diff
@@ -35,6 +36,11 @@ int main(int argc, const char *argv[]) {
                 o->extra["output"] = argument;
                 Colors::disable();
               })
+      .add("--metrics", "", "prints binary metrics",
```

Comment:
Or perhaps we should wait with that for a future refactoring of binaryen-shell that can load and store binaries.